pccommon/pccommon/tracing.py

```diff
@@ -25,6 +28,11 @@
 logger = logging.getLogger(__name__)
 
 
+COLLECTION = "spatio.collection"
```

Comment:
I don't think we want to use these "spatio" labels as keys for the span, down in the `trace_request` method we just use `collection` and `item`, and should probably keep that consistent for the search endpoint as well.

```python
if collection_id is not None:
  tracer.add_attribute_to_current_span(
      attribute_key="collection", attribute_value=collection_id
  )
  if item_id is not None:
      tracer.add_attribute_to_current_span(
          attribute_key="item", attribute_value=item_id
      )
```